packages/web/src/pages/artist-coins-launchpad-page/pages/BuyCoinPage.tsx

```diff
@@ -45,7 +45,7 @@ const messages = {
   youReceive: 'You Receive',
   valueInUSDC: 'Value',
   hintMessage:
-    "Buying shares now makes sure you can get in at the lowest price before others beat you to it. You'll still receive your vested coins over time after your coin reaches it's graduation market cap (500K AUDIO).",
+    "Buying shares now makes sure you can get in at the lowest price before others beat you to it. You'll still receive your vested coins over time after your coin reaches it's graduation market cap (1M AUDIO).",
```

Comment:
Update to

Buying now makes sure you can get in at the lowest price before others beat you to it. You'll still receive your vested coins over time after your coin reaches its graduation market cap (1M $AUDIO).


per latest design